samples/apps/copilot-chat-app/webapp/src/index.tsx

```diff
@@ -33,6 +33,12 @@ document.addEventListener('DOMContentLoaded', async () => {
         var msalInstance: IPublicClientApplication | null = null;
         if (validEnvFile) {
             msalInstance = new PublicClientApplication(AuthHelper.msalConfig);
+
+            msalInstance.handleRedirectPromise().then((response) => {
+                if (response) {
```

Comment:
What's the behavior if the log-in fails? Is it still showing a banner alert?

Reply:
If login fails, user stays on MSFT account login page. If they manually redirect to Copilot Chat, they will still see the Copilot Chat login view.